src/App.tsx

```diff
@@ -31,6 +47,20 @@ export default function App() {
             path="/activities"
             element={<ActivitiesList activities={ACTIVITIES_MOCK} />}
           />
+          <Route
+            path="/signup"
+            element={<AccountCreation onAccountCreate={handleAccountCreate} />}
```

Comment:
Really great job in this PR overall with using `on-` vs `handle-` prefixes accurately for the event handlers!